crates/esplora/tests/blocking_ext.rs

```diff
@@ -239,14 +239,14 @@ pub fn test_update_tx_graph_gap_limit() -> anyhow::Result<()> {
         sleep(Duration::from_millis(10))
     }
 
-    // A scan with gap limit 4 won't find the second transaction, but a scan with gap limit 5 will.
+    // A scan with gap limit 4 won't find the second transaction, but a scan with gap limit 6 will.
```

Comment:
Same as above: it makes no sense  to check with `stop_gap=n` and then with `stop_gap=n+2`. You want to check that the stop_gap is exactly n (in this case, 6), and that n-1 doesn't work. 

This should be:
stop_gap of 5: doesn't find
stop_gap of 6: finds